models/AmazonS3.cfc

```diff
@@ -888,7 +888,8 @@ component accessors="true" singleton {
 	struct function downloadObject(
 		required string bucketName = variables.defaultBucketName,
 		required string uri,
-		required string filepath
+		required string filepath,
+		boolean getAsBinary = false
```

Comment:
Can you change to using `yes` and `no` due to Adobe being dumb on booleans please.

Reply:
All set